Source/DeepLearning/WebModel.h

```diff
@@ -105,14 +105,15 @@ class WebWave2Wave : public Model, public Wave2Wave {
             .getChildFile("control_spec.json");
     outputPath.deleteFile();
 
-
-    // juce::File scriptPath = juce::File::getSpecialLocation(
-    //     juce::File::currentApplicationFile
-    // ).getChildFile("Contents/Resources/gradiojuce_client/gradiojuce_client");
-
-    juce::File scriptPath = juce::File::getSpecialLocation(
+    #if JUCE_MAC
+      juce::File scriptPath = juce::File::getSpecialLocation(
         juce::File::currentApplicationFile
-    ).getParentDirectory().getChildFile("gradiojuce_client/gradiojuce_client.exe");
+      ).getChildFile("Contents/Resources/gradiojuce_client/gradiojuce_client");
+    #else
+      juce::File scriptPath = juce::File::getSpecialLocation(
+          juce::File::currentApplicationFile
+      ).getChildFile("../../Resources/dist/gradiojuce_client/gradiojuce_client");
+    #endif
```

Comment:
solid, but the `command` string below still has the windows-only `start` invocation. when i was playing around with this a few days ago i'd written a separate function to return a platform-specific command invocation, something like
```
std::string gradioCommand()
{
    #if JUCE_WINDOWS
      // gradio command with "start," though this may be tied to the `PYTHONIOENCODING=UTF-8` stuff
    #else
      // unprefixed non-exe string
    #endif
}

Reply:
whichever route you go with this, the `process` method has the same dot exe issue

Reply:
fixing both instances of the dot exe issue and making sure that `start` wasn't called in `load` was enough to run a successful test for me

Reply:
I added a helper function to get the script path, so the same logic is used for both methods.